trailblazer/io/controller.py

```diff
@@ -18,10 +18,7 @@ class ReadFile:
     @classmethod
     def get_content_from_file(cls, file_format: str, file_path: Path) -> Any:
         """Read file using file format dispatch table."""
-        try:
-            return cls.read_file[file_format](file_path=file_path)
-        except FileNotFoundError as error:
-            raise FileNotFoundError(f"File not found: {file_path}") from error
+        return cls.read_file[file_format](file_path=file_path)
```

Comment:
Won't this risk stopping a TB scan if a file is missing?

Reply:
No, the catch all below remains! I added this line in my last PR

Reply:
👍